chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts

```diff
@@ -8,10 +8,12 @@ export class MockCustomerService {
   constructor() { }
   public generateCustomer() {
     // TODO: create a new customer.
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
     const newCust = {
```

Comment:
Rename `newCust` variable to `newCustomer`.

Reply:
Changed name from newCust to newCustomer.